buildconfig/CMake/Eigen.cmake

```diff
@@ -8,7 +8,8 @@ option(USE_SYSTEM_EIGEN "Use the system installed Eigen - v${eigen_version}?" OF
 
 if(USE_SYSTEM_EIGEN)
   message(STATUS "Using system Eigen")
-  find_package(Eigen ${eigen_version} EXACT REQUIRED)
+  find_package(Eigen3 ${eigen_version} EXACT REQUIRED
```

Comment:
I have no strong preference either way. As you say, I think it is unlikely to be used in this way anyway.